index.html

```diff
@@ -0,0 +1,88 @@
+                            <li>Melhorar a capacidade de escrita, especialmente a ecessidade de síntase.</li>
+                            <li>Ser grato por cada pequena conquista.</li>
+                        </ol>
+                    </nav>
+
+                    <nav>
+                        <h2>Lições a aprender</h2>
+                        <ul id="licoes_a_aprender">
+                          <li>Não desistir no primeiro problema.</li>
+                          <li>Acreditar que todos são capazes, especialmente você.</li>
+                          <li>Aproveitar o tempo de forma sábia</li>
+                          <li>Perder a vergonha de expor opiniões e mostrar as minhas vulnerabilidades.</li>
+                          <li>Aprender de forma satisfatórias as linguagens propostas.</li>
+                          <li>Desenvolver minha inteligência emocional.</li>
+                          <li>Viver um dia de cada vez.</li>
+                          <li>Agradecer sempre, especialmente quando der errado, mas te ensinar algo.</li>
+                          <li>Buscar evoluir sempre e no seu ritmo.</li>
+                          <li>Praticar a empatia com as pessoas ao meu redor.</li>
+                        </ul>
+                    </nav>
+                    <table class="table">
+                        <tr>
+                            <th>Situação</th>
+                            <th>Habilidade</th>
+                        </tr>
+                        <tr>
+                            <td>Erro</td>
+                            <td>Colaboração</td>
+                        </tr>
+                        <tr>
+                            <td>Conflito</td>
+                            <td>Inteligência emocional</td>
+                        </tr>
+                    </table>
+                </section>
+
+                <article class="article lado-esquerdo">
+                    <h2 class="alinha-titulo">Meu aprendizado</h2>
+                    <figure>
+                        <img id="minha_foto" class="alinha-foto" src="foto.jpg" alt="Foto da programadora.">
+                    </figure>
+                    <p class="limpa">"Quem diria que em apenas duas semanas fossem acontecer tantas transformações na minha vida. Se alguém me falasse que ao entrara na Trybe eu reconheceria a missão e a visão de forma tão acolhedora, e que essa ação me traria paz. As Hards skllis são meu novo desafio diário, mas hoje posso dizer que começa a fazer sentindo algumas questões de Git e Github, HTML e CSS. É uma grande vitória!"</a></p>
+                </article>
+                
+                <aside class="limpa">
+                    <h2>Aprendizados</h2>
+                    <p>As Hards skllis são meu novo desafio diário, mas hoje posso dizer que começa a fazer sentindo algumas questões de Git e Github, HTML e CSS. É uma grande vitória! 
```

Comment:
@marianasaraiva acredito que tenha um erro tipográfico neste item. 
```suggestion
                    <p>As Hard skills são meu novo desafio diário, mas hoje posso dizer que começa a fazer sentindo algumas questões de Git e Github, HTML e CSS. É uma grande vitória! 
```